packages/design-system/src/Documentation/components/CodeBlock.tsx

```diff
@@ -15,16 +15,14 @@ function CodeBlock({ code }: CodeBlockProps) {
         toast.show("Copied to clipboard", { kind: "success" });
       }}
     >
-      <Tooltip content="Click here to copy" mouseEnterDelay={0.7}>
-        <Code>{code}</Code>
-      </Tooltip>
+      <Code>{code}&nbsp;</Code>
+      <Icon name="copy-control" size="md" />
```

Comment:
As mentioned in the issue description, please use **Icon Button** and not Icon component.
https://design-system.appsmith.com/?path=/story/design-system-button--icon-button-story&args=startIcon:copy-control;kind:tertiary

Reply:
Made the necessary changes @albinAppsmith , please review and lmk if anything else needs to be changed, thanks!